sample-root-scripts/MakeKin.py

```diff
@@ -21,7 +20,7 @@
              "Cylinder_60x74_20inchBandL_14perCent":[7400./2., 6000.],
              "Cylinder_60x74_20inchBandL_40perCent":[7400./2., 6000.]}
```

Comment:
```suggestion
             "Cylinder_60x74_20inchBandL_40perCent":[7400./2., 6000.],
             "HyperK":[7080./2., 5480.]}
```
These numbers are taken from https://github.com/WCSim/WCSim/blob/7ee6f09b8d8fa26d179ffb7a9b22571e172286cc/src/WCSimDetectorConfigs.cc#L222-L223